care/facility/models/summary.py

```diff
@@ -0,0 +1,20 @@
+from uuid import uuid4
+
+from django.core.validators import RegexValidator
+from django.db import models
+
+from care.facility.models.facility import Facility
+from django.contrib.postgres.fields import JSONField
```

Comment:
Codacy found an issue: [third party import "from django.contrib.postgres.fields import JSONField" should be placed before "from care.facility.models.facility import Facility"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)